.travis.yml

```diff
@@ -24,10 +24,9 @@ branches:
 
 env:
   - EMBER_TRY_SCENARIO=ember-default
-  - EMBER_TRY_SCENARIO=ember-lts-2.4
-  - EMBER_TRY_SCENARIO=ember-lts-2.8
```

Comment:
Why did you remove this?

Reply:
Not manually removed, ember init changed this

Reply:
Ember-source 2.4 is no longer available